cfme/tests/containers/test_containers_default_views.py

```diff
@@ -23,7 +23,7 @@
     'Containers Providers': 'containers_providers',
     'Projects': 'containers_projects',
     'Routes': 'containers_routes',
-    # 'Nodes': 'containers_nodes',
+    'Nodes': 'containers_nodes',
```

Comment:
This was removed because the nav branch for containers.node has been removed. The force_navigate call in test_containers_providers_default_view will fail.

Once all of the objects in this map support navmazing destinations, we'll change the force_navigate to a navigate_to() and update this map accordingly to provide the proper objects.